ansible_ai_connect/ai/api/views.py

```diff
@@ -1117,8 +1118,7 @@ class ChatEndpointThrottle(EndpointRateThrottle):
         scope = "chat"
 
     permission_classes = [
-        permissions.IsAuthenticated,
-        IsAuthenticatedOrTokenHasScope,
+        IsRHEmployee | IsTestUser,
```

Comment:
Guys, this means a token scoped session only cannot longer the chat API, is that correct? May it break integration tests?
@jameswnl @goneri @TamiTakamiya

Reply:
Please keep exiting two `permission_classes`
```
        permissions.IsAuthenticated,
        IsAuthenticatedOrTokenHasScope,
```
The first `permissions.IsAuthenticated` is actually redundunt since both `IsRHEmployee` and `IsTestUser` checks `user.is_authenticated` internally, but I think we'd better keep them to be consistent with other views and the redundancy is negligible.

Reply:
fixed, thanks!